src/data/teamData.ts

```diff
@@ -1,11 +1,131 @@
-import Seamus_Brugh from "../Assets/team/Seamus Brugh.png";
-const team = [
+
+type teamMember = {
```

Comment:
I think custom types and interfaces are customarily capitalized. So instead it would be `TeamMember`

Reply:
Ah good catch.